addon/services/infinity.js

```diff
@@ -225,6 +199,10 @@ export default Service.extend({
     // chunk requests by indicated perPage param
     const perPage = options.perPage || 25;
 
+    // store service methods (defaults to ember-data if nothing passed)
+    const store = options.store || get(this, 'store');
```

Comment:
getting it off the service is necessary b/c the infinity model is not created by a container (thus can't inject anything)